twisted_tools/buildbot.py

```diff
@@ -3,6 +3,7 @@
 import treq
 import twisted
 
+BASE_URL = 'http://buildbot.twistedmatrix.com'
```

Comment:
I should note that we have recently made the TLS certificate for `buildbot.twistedmatrix.com` valid and correct.